akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -113,14 +112,14 @@ officially supported. If you're on a Big Endian processor, such as Sparc, it is
 
 ## Migrating from classic remoting
```

Comment:
I think we should have all migration in one place, in the migration guide, so move relevant parts of this to the migration guide

entire section can be removed from here

Reply:
:+1:

Reply:
pending?

Reply:
lol I did it the other way around.